src/views/EventDetails.vue

```diff
@@ -1,65 +1,74 @@
 <template>
-<div>
-    <div v-if="todoEvent" class="event__details">
-        <div class="event__id"><h1>Event</h1>{{"#"+todoEvent.id}}</div>
-        <div class="event__title" >{{" " +todoEvent.title}}</div>
-        <div><span>To do:</span>{{" " +todoEvent.description}}</div>
-        <div><span>Where: </span>{{" " +todoEvent.location }}</div>
-        <div><span>When: </span>{{" " +todoEvent.date }}</div>
-        <div><span>Time:</span>{{" " +todoEvent.time }}</div>
-        <div><span>Organizer:</span>{{" " +todoEvent.organizer}}</div>
-        <div><span>Category:</span>{{" " +todoEvent.category}}</div>
-        <div><span>Attendees:</span>{{" " + attendees()}}</div>
-    </div>
+  <div
+    v-if="targetEvent"
+    class="event__details"
+  >
+    <div class="event__id"><h1>Event</h1>{{ '#' + targetEvent.id }}</div>
+    <div class="event__title">{{ ' ' + targetEvent.title }}</div>
+    <div><span>To do:</span>{{ ' ' + targetEvent.description }}</div>
+    <div><span>Where: </span>{{ ' ' + targetEvent.location }}</div>
+    <div><span>When: </span>{{ ' ' + targetEvent.date }}</div>
+    <div><span>Time:</span>{{ ' ' + targetEvent.time }}</div>
+    <div><span>Organizer:</span>{{ ' ' + targetEvent.organizer }}</div>
+    <div><span>Category:</span>{{ ' ' + targetEvent.category }}</div>
+    <div><span>Attendees:</span>{{ ' ' + attendees() }}</div>
   </div>
-  
 </template>
 
 <script>
-import store from '../mocks/store';
```

Comment:
It's better to have a single source of truth rather than import it everywhere. So pass the events as props